docs/vi/week01/01.md

```diff
@@ -0,0 +1,22 @@
+---
+lang: vi
+lang-ref: 01.md
+title: week01
```

Comment:
Title needs translation.

Reply:
I want to use this format (the same English version)

Reply:
If it needs, I will translate it. Please advise!

Reply:
Use "Tuần 01" instead.

Reply:
Ok Sir. I corrected.